README.rst

```diff
@@ -58,9 +58,17 @@ The API is pretty simple, three functions are provided in the ``DNSResolver`` cl
 Note for Windows users
 ======================
 
-This library requires the asyncio loop to be a `SelectorEventLoop`, which is not the default on Windows.
+This library requires the use of an ``asyncio.SelectorEventLoop`` on Windows
+**only** when using a custom build of ``pycares`` that links against a system-
+provided ``c-ares`` library **without** thread-safety support. This is because
+non-thread-safe builds of ``c-ares`` are incompatible with the default
+``ProactorEventLoop`` on Windows.
 
-The default can be changed as follows (do this very early in your application):
+If you're using the official prebuilt ``pycares`` wheels (version 4.7.0 or
```

Comment:
Probably clear, but I guess the ones on PyPI are considered the official prebuilt ones?

Reply:
tweaked in 36ce0f351fb528e61d70fd1b25ba2db2773c31d7